perma_web/static/css/style-responsive.scss

```diff
@@ -5862,4 +5862,15 @@ li[data-read_only="true"] {
 Example: calc(4rem / 16) is equal to 4px or 0.25rem */
 .u-pb-150 {
   padding-bottom: calc(18rem / 16);
+}
+
+/* CSS transition classes automatically added/removed by Vue */
+.fade-enter-active,
+.fade-leave-active {
+  transition: opacity 0.3s ease;
+}
+
+.fade-enter-from,
+.fade-leave-to {
+  opacity: 0;
 }
```

Comment:
The hyphen-based namespacing here is required for Vue to be able to add and remove these classes to trigger CSS transitions.